include/pa_asio.h

```diff
@@ -125,7 +194,7 @@ PaError PaAsio_SetStreamSampleRate( PaStream* stream, double sampleRate );
 typedef struct PaAsioStreamInfo{
     unsigned long size;             /**< sizeof(PaAsioStreamInfo) */
     PaHostApiTypeId hostApiType;    /**< paASIO */
-    unsigned long version;          /**< 1 */
+    unsigned long version;          /**< 2 */
 
     unsigned long flags;
```

Comment:
Use the flags field to indicate whether a feature in the struct is to be used. There is currently a flag called `paAsioUseChannelSelectors`. A new flag `paAsioUseMessageCallback` should be added, and that should be used rather than struct size+version, or NULL to indicate presence of the `messageCallback`. That's the way that we do it elsewhere and I think it's best to be consistent here.

Reply:
Done.  Make note of the validation code.